src/mmw/sass/components/_tabs.scss

```diff
@@ -63,29 +63,29 @@
 //Model Output Tabs
 .output-tabs-wrapper {
   .nav-tabs {
-    font-weight: 500;
-    font-size: 0.56rem;
+    font-weight: 400;
+    font-size: $font-size-h5;
     border-bottom: 1px solid $black-12;
 
     li a {
       border-radius: 0!important;
       border-top: none!important;
       border-left: none!important;
       border-right: none!important;
-      text-transform: uppercase;
     }
 
     li > a {
       color: $ui-secondary;
-      padding: 0 0.85rem;
-      line-height: 34px;
+      padding: 0 0.7rem;
+      line-height: 42px;
       background-color: transparent;
-      height: $sidebar-tab-subheader-height;
```

Comment:
Maybe we should rename this to `$analyze-tab-height` now that it's only used in the analyze stage.